packages/bot-web-ui/src/components/summary/summary-card.jsx

```diff
@@ -104,12 +123,17 @@ SummaryCard.propTypes = {
 };
 
 export default connect(({ summary_card, common, run_panel, ui }) => ({
-    contract: summary_card.contract,
+    addToast: ui.addToast,
+    contract_info: summary_card.contract_info,
+    contract_store: summary_card,
```

Comment:
Don't you miss something here? `summary_card.contract_store`?
Although summary card store doesn't have `contract_store`

Reply:
Actually no, it's the instance of summary_card store itself.